src/components/OpenSource/WhyOpenSource.js

```diff
@@ -0,0 +1,27 @@
+import React, { Fragment } from 'react'
+import SubtitleWithBody from '../Common/SubtitleWithBody'
+import styled from 'styled-components'
+
+const StyledSubtitleWithBody = styled(SubtitleWithBody)`
```

Comment:
I believe the padding is already correct on `SubtitleWithBody` & if you remove these lines, then the component in storybook will look the same anyway.

Besides, why do we need extra spacing on the `SubtitleWithBody` component?

![image](https://user-images.githubusercontent.com/15656538/58550019-c4845e00-8204-11e9-92e2-700e65205a40.png)